### PR TITLE
claimfey.com

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/115641

<details><summary>Screenshot:</summary>

Снимок экрана 2022-04-15 в 14 47 41

</details><br/>

### CODE REVIEW

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -781,6 +781,8 @@ btcbunch.com#%#//scriptlet("abort-on-property-write", "TestAd")
 @@||googletagmanager.com/gtag/js$domain=btcbunch.com
 @@||static.surfe.pro/js/net.js$domain=btcbunch.com
 !#endif
+! https://github.com/AdguardTeam/AdguardFilters/issues/115641
```
Comment (Alex-302) [inline EnglishFilter/sections/antiadblock.txt:784]: We already have the rules for this script, just add new domain to the necessary scriptlet and URL rules
https://github.com/AdguardTeam/AdguardFilters/blob/58fd6094784a6d77e66e3ac309b496d3c8bc1ad2/EnglishFilter/sections/antiadblock.txt#L567

Comment has the word which helps to find the rules easily
image

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/antiadblock.txt [hunk 2]
```diff
@@ -781,6 +781,8 @@ btcbunch.com#%#//scriptlet("abort-on-property-write", "TestAd")
...
+@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:785]: `||claimfey.com/cryptonews/wp-content/uploads/bqanwcfp.js`
You should have found the initiator script.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/antiadblock.txt (hunk 1 above)
Comment (zloyden) [reply EnglishFilter/sections/antiadblock.txt:784]: Yep, but it's not necessary here.

File: EnglishFilter/sections/antiadblock.txt [hunk 3]
```diff
@@ -571,9 +571,9 @@ pngtosvg.com,vladan.fr,unityassets4free.com,clockks.com#%#//scriptlet("abort-on-
...
...rgemoncada.com|hollywoodsmagazine.com|anicosplay.net|ewybory.eu|nanoroms.com|egymanga.net|flipreview.com|hiphopa.net|pngtosvg.com|compartilhandobr.com|loksado.com|ecoursefree.com|daemon-hentai.com|yigeplus.xyz|cgchotbox.com|launcherfull.com|cheatbyte.com|techycoder.com|arasindakifark.net|kitguru.net|gsmxt.com|mangaxp.com|technicalatg.com|technicalatg.xyz|freetutorialsdownload.online|tutsgalaxys.com|vipmods.net|freetutorialsudemy.com|adonisfansub.com|getfreecourses.org|themovieslab.in|bitcoinegypt.news|talkjarvis.com|satcesc.com|dongknows.com|getfreecourses.net|citychilli.com|livenewsof.com|pinayvids.com|hardcoregames.ca|pasend.*|unityassetcollection.com|crypto-fun-faucet.de|fileoops.com|samuraiscan.com|3dmodelshare.org|cgtips.org|dailyvideoreports.net|codesnail.com|ma-x.org|sitarchive.com|onlyfanesse.com|codingshiksha.com|graphicux.com|w3layouts.com|musike.net|windowsmatters.com|sonixgvn.net|avoiderrors.com|gatcha.org|sayrodigital.com|justandroid.net|4horlover.com|topnewsshow.com|3dzip.org|ovagames.com|lewdsenpai.com|mitsmits.com|asyadrama.com|r3ndy.com|mylivewallpapers.com|e-sanatatea.ro|ourcoindigibyte.xyz|ourcoindash.xyz|ourcoineth.xyz|ourcoinfaucet.xyz|ourcoin.xyz|latestjav.com
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt]: This is not necessary, see my comment https://github.com/AdguardTeam/AdguardFilters/pull/115643#discussion_r851230839

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**